contents/stats.md

```diff
@@ -0,0 +1,445 @@
+# Statistics {#sec:stats}
+
+This content was created in order to awaken the reader to the importance of statistics for science and knowledge generation.
```

Comment:
```suggestion
An important part of data science is being able to make predictions.
In essence, the world is constantly generating data and we're only seeing the small part of the data that we captured.
This means that we have to take this small part and infer things about the real world.
This is called statistics.
```
Something like this? I avoid "awaken" because I don't think that a typical reader of our book would not already know the importance of statistics.

Reply:
Statistics is not only predictions. We need to say more about "inference" since our audience are PhD candidates from all fields of applied sciences. That's why I've inserted "importance of statistics for science and knowledge generation"

Reply:
In essence, when you infer a model from a dataset, you're gonna use that model to make predictions. I'll leave it up to you what you decide to write here. :+1: